README.md

```diff
@@ -1,6 +1,23 @@
 # COBalD/TARDIS Plugins for High Energy Physics
 
-This repository collects official and third-party plugins.
+[![Documentation Status](https://readthedocs.org/projects/cobald-hep-plugins/badge)](https://cobald-hep-plugins.readthedocs.io)
+
+This package collects official and third-party plugins.
+
+## Do I use these plugins?
```

Comment:
What is the focus you want to put here? Either *How do I use these plugins?* or *Do I want to use these plugins?*. The first sounds better from my point of view :)

Reply:
Adjusted it to "How do I use these plugins?".